src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -42,9 +41,4 @@
 
 register_cli_argument('acr create', 'registry_name', completer=None)
 register_cli_argument('acr create', 'resource_group_name',
-                      validator=validate_resource_group_name)
-
-register_cli_argument('acr', 'enable_admin', help='Enable admin user',
-                      validator=validate_admin_user)
-register_cli_argument('acr', 'disable_admin', help='Disable admin user',
-                      validator=validate_admin_user)
+                      validator=validate_resource_group_name)
```

Comment:
Should have a newline at the end of the file. Surprised pylint didn't complain loudly.